vmms/ec2SSH.py

```diff
@@ -102,6 +110,13 @@ def instanceName(self, id, name):
         """
         return "%s-%d-%s" % (config.Config.PREFIX, id, name)
 
+    def keyPairName(self, id, name):
```

Comment:
Yes. You are right. They are the same. Frankly speaking, I just copied the `instanceName()` function. The only difference is that `instanceName()` is used to name the instance, but `keyPairName()` is used to name the keypair associated with the instance. If you believe that this makes the code redundant, I can delete it.